### ISSUE TITLE
Search

### ISSUE THREAD

**Issue by dessalines** [dessalines/jerboa#27]

**Comment by twizmwazin** [comment]
I think this encompasses all search features currently exposed in the web UI. Anything not supported there is likely unsupported in the API, and an issue would need to be raised and implemented on the server before we can do anything here.

**Comment by bhcompy** [comment]
Conceptually, though, I'm not really asking to search through the API, rather just search what is returned within the application, similar to ctrl-f through a desktop (which also doesn't rely directly on API/HTTP calls, just searches on already returned and formatted data).

**Comment by twizmwazin** [comment]
I see. I think that would have a lot edge cases to consider, like not all comments being automatically loaded on big threads where this would be most useful, but doesn't seem impossible. It is definitely out of scope of this current issue, however.

**Comment by marsara9** [comment]
I can create a separate issue for this, but what about just using my https://www.lemmy-search.com/ ?  i.e. have the search feature in jabora use my APIs as an alternative to the built-in Lemmy API.

I currently only support searching posts and communities but users may come soon-ish.

This search engine also allows for more complex queries than what is currently supported inside Lemmy. 

More details here: https://www.github.com/marsara9/lemmy-search
